Logger.Tests/LogFactoryTests.cs

```diff
@@ -6,6 +6,25 @@ namespace Logger.Tests
     [TestClass]
     public class LogFactoryTests
     {
-        
+        [TestMethod]
+        public void LogFactory_SetsClassName()
+        {
+         // Arrange
+         LogFactory myLogFactory = new LogFactory();
+         myLogFactory.configureFileLogger("foobar");
+
+         FileLogger myLogger = (FileLogger)myLogFactory.CreateLogger(nameof(LogFactoryTests));
```

Comment:
Looks great! No need to cast to a file logger, you can simply create a new BaseLogger.
```suggestion
         BaseLogger myLogger = myLogFactory.CreateLogger(nameof(LogFactoryTests));
```